.env.compose

```
@@ -557,7 +557,45 @@ MCP_WS_TLS=false
-MCP_TRUSTED_PROXIES=loopback,linklocal,uniquelocal
+MCP_WS_TRUSTED_PROXIES=loopback,linklocal,uniquelocal
+
+# OAuth 2.0 Authorization Configuration (Production)
+MCP_AUTH_ENABLED=true
+MCP_AUTH_RESOURCE_URI=https://mcp.gauzy.co
+MCP_AUTH_REQUIRED_SCOPES=mcp.read,mcp.write,mcp.admin
+
+# JWT validation for production (using RS256 with public key)
+MCP_AUTH_JWT_AUDIENCE=https://mcp.gauzy.co
+MCP_AUTH_JWT_ISSUER=https://auth.gauzy.co
+MCP_AUTH_JWT_ALGORITHMS=RS256,ES256
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_AUTH_JWT_ALGORITHMS key should go before the MCP_AUTH_JWT_AUDIENCE key

(UnorderedKey)